.github/checkInstallTime.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const yarnOut = fs.readFileSync(0, {encoding: 'utf8'})
+
+const [installTimeString] = /(?<=^Done in )\d+\.\d+(?=s\.$)/m.exec(yarnOut)
+const installTime = Number(installTimeString)
+
+console.log(`Install time: ${installTime}s`)
+
+if (installTime < 30) {
```

Comment:
We'll have to account for CI installations being faster than local ones. Do y'all think we should leave it at < 30 green / < 50 orange | >= 50 red or lower our thresholds?

Reply:
I would be thinking we probably just want to know how large the delta is.

Reply:
That doesn't seem trivial to do. Can you point me towards an implementation route?

Reply:
Yeah that would be an ideal solution. This will be ok for now but will not age well as it will change over time and need revisiting unless we can somehow store previous vals in state somewhere. Perhaps we can us a gh actions addon (perhaps something similar to node_modules caching) to give us a file system to store previous values.

Reply:
Tbh, that actually seems pretty crazy - but I don't think we're the only ones having the need for such a tool, so maybe there's already something in existance.